spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb

```diff
@@ -62,6 +62,16 @@
       it 'includes the veterans birthdate' do
         expect(subject).to include 'DOB: 10-10-1968'
       end
+
+      context 'when veteran birthdate includes timestamp artifact' do
+        let(:patient_name) do
+          { first: 'Cat', middle: 'Marie', last: 'Power', suffix: 'Jr.', birthdate: '1968-10-10T00:00:00+00:00' }
```

Comment:
Use a day that is different from the month and obvious that it is the day (i.e., there's no month 20):
```suggestion
          { first: 'Cat', middle: 'Marie', last: 'Power', suffix: 'Jr.', birthdate: '1968-10-20T00:00:00+00:00' }
```